### ISSUE TITLE
Help for Users

### ISSUE THREAD

**Issue by ocgabs** [OceanParcels/parcels#790]
Hi, 
I am trying to get PARCELS working for a new NEMO configuration. I am using the simplest possible script for testing:

```python
from parcels import FieldSet, ParticleSet, ScipyParticle, JITParticle, AdvectionRK4, AdvectionRK4_3D, ParticleFile, Variable
from argparse import ArgumentParser
import numpy as np
import pytest
from glob import glob
from datetime import timedelta as delta
from os import path
import time
from netCDF4 import Dataset

data_path = '/projectsa/anchor/NEMO/AMM60/FebLinks/'
ufiles = sorted(glob(data_path+'2012_Feb_*.nc'))
vfiles = sorted(glob(data_path+'2012_Feb_*.nc'))
grid_file = '/projectsa/anchor/NEMO/AMM60/mesh_mask.nc'
filenames = {'U': {'lon': grid_file,'lat': grid_file,'data': ufiles},
             'V': {'lon': grid_file,'lat': grid_file,'data': vfiles},}
variables = {'U': 'uo', 'V': 'vo'}
dimensions = {'lon': 'glamf', 'lat': 'gphif','time': 'time_counter' }
field_set = FieldSet.from_nemo(filenames, variables, dimensions)
pset = ParticleSet.from_list(field_set, pclass=JITParticle, lon=[1.7, 0.3], lat=[59.34, 57.19])
pfile = ParticleFile("Yo_AMM60_5seeds", pset, outputdt=delta(hours=0.5))
#k_adv3D = pset.Kernel(AdvectionRK4_3D)
k_adv = pset.Kernel(AdvectionRK4)
pset.execute(k_adv, runtime=delta(days=1), dt=delta(minutes=10),output_file=pfile)
```
And I am getting the following error... 
Thanks in advance for your help!

```
(/login/gmaya/.conda/envs/py3_parcels)livljobs7 PARCELS $ python 2D_AMM60_Parcels.py
WARNING: Casting depth data to np.float32
INFO: Compiled JITParticleAdvectionRK4 ==> /tmp/parcels-18684/7dbd2d672682b91815628ff1324315bb.so
Traceback (most recent call last):
  File "2D_AMM60_Parcels.py", line 29, in <module>
    pset.execute(k_adv, runtime=delta(days=1), dt=delta(minutes=10),output_file=pfile)
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/parcels/particleset.py", line 389, in execute
    next_input = self.fieldset.computeTimeChunk(time, np.sign(dt))
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/parcels/fieldset.py", line 601, in computeTimeChunk
    f.computeTimeChunk(data, tind)
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/parcels/field.py", line 929, in computeTimeChunk
    if len(filebuffer.data.shape) == 2:
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/parcels/field.py", line 1450, in data
    data = data[ti, self.indices['depth'], self.indices['lat'], self.indices['lon']]
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/xarray/core/dataarray.py", line 479, in __getitem__
    return self.isel(indexers=self._item_key_to_dict(key))
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/xarray/core/dataarray.py", line 827, in isel
    ds = self._to_temp_dataset().isel(drop=drop, indexers=indexers)
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/xarray/core/dataset.py", line 1646, in isel
    new_var = var.isel(indexers=var_indexers)
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/xarray/core/variable.py", line 927, in isel
    return self[tuple(key)]
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/xarray/core/variable.py", line 636, in __getitem__
    data = as_indexable(self._data)[indexer]
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/xarray/core/indexing.py", line 630, in __getitem__
    return type(self)(_wrap_numpy_scalars(self.array[key]))
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/xarray/core/indexing.py", line 607, in __getitem__
    return type(self)(_wrap_numpy_scalars(self.array[key]))
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/xarray/core/indexing.py", line 520, in __getitem__
    return type(self)(self.array, self._updated_key(indexer))
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/xarray/core/indexing.py", line 491, in _updated_key
    full_key.append(_index_indexer_1d(k, next(iter_new_key), size))
  File "/login/gmaya/.conda/envs/py3_parcels/lib/python3.7/site-packages/xarray/core/indexing.py", line 291, in _index_indexer_1d
    indexer = _expand_slice(old_indexer, size)[applied_indexer]
IndexError: index 766 is out of bounds for axis 0 with size 766
```

**Comment by erikvansebille** [comment]
Thanks for reporting @ocgabs. I'm not really sure what triggers this error, but it might be related to some problem with the chunking in `dask`. We've yesterday released a new version of Parcels (v2.1.5) on conda, which has much improved chunking. Could you try rerunning with that latest version, and see if that fixed your problem? if not, could you share a snippet of the netcdf data, so that I can have a look myself?

**Comment by ocgabs** [comment]
Hi again, 
I am wondering if you already have some kernels specific for tracking plastics and what are their capabilities? We are developing a project proposal and it will be great to have this information. Thanks in adavance.

**Comment by erikvansebille** [comment]
Closing, since assuming that the error was fixed in v2.1.5
